.github/workflows/tagged_master_release.yml

```diff
@@ -99,6 +107,33 @@ jobs:
           name: agave-win
           path: ./build/agave-win.zip
 
+  publish_pythonclient:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: "recursive"
+      - name: Install Python 3.9 version
+        uses: actions/setup-python@v1
+        with:
+          python-version: "3.9"
+          architecture: "x64"
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel
+      - name: Build Package
+        run: |
+          cd agave_pyclient
+          python setup.py sdist bdist_wheel
+      - name: Publish to PyPI
+        uses: pypa/gh-action-pypi-publish@master
```

Comment:
For third-party actions, GitHub recommends pinning to a SHA:

https://docs.github.com/en/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions

(and by its nature, this action gets passed sensitive information).

Reply:
Good idea, will at least pin to a known release tag